src/app/services/events.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { StompService } from 'ng2-stomp-service';
+import { environment } from '../../environments/environment';
+import { Event } from '../models/event';
+import { EventType } from '../models/eventType';
+
+@Injectable()
+export class EventsService {
+
+  constructor(private stomp: StompService) {
+    // Configure stomp client
+    stomp.configure({
+      host: environment.sf_server_address + '/events',
+      debug: environment.stomp_debug,
+      queue: { 'init': false, 'events': false }
+    });
+
+    // Start socket connection
+    stomp.startConnect().then(() => {
+      stomp.done('init');
+      console.log('Da client has konnected');
+    });
+  }
+
+  sendAddTrackEvent(user: string, trackname: string, artist: string) {
+    const addTrackEvent = new Event(user, EventType.ADDTRACK);
+    addTrackEvent.setContext('artist', artist);
+    addTrackEvent.setContext('trackname', trackname);
+    this.stomp.send('/app/events', addTrackEvent);
```

Comment:
This way of sending stuff suggest that events are being stored globally not per room.

/app/room/{id}/events 

makes more sense i think

Reply:
Yes, but I'm not taking into account room as they are not coded yet. But I agree this will need to be refactored to take into account rooms. It's just too early to think about it